### PR TITLE
Settings: Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by JohnsonRan**

### CODE REVIEW

File: core/tests/batterystatstests/BatteryStatsViewer/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     SPDX-FileCopyrightText: 2023 The Calyx Institute
+     SPDX-License-Identifier: Apache-2.0
+-->
+<resources>
+    <string name="app_name">电池统计查看器</string>
+    <string name="header">电池统计</string>
+    <string name="display_dashboard_summary">查看详细的电池用量统计</string>
+</resources>
```
Comment (idoybh) [inline core/tests/batterystatstests/BatteryStatsViewer/res/values-zh-rCN/strings.xml:10]: this file won't compile. just for being there. need to find someplace else to translate these

Comment (JohnsonRan) [reply core/tests/batterystatstests/BatteryStatsViewer/res/values-zh-rCN/strings.xml:10]: eh, `packages/SystemUI/res/values-zh-rCN/yaap_strings.xml` here maybe?

Comment (idoybh) [reply core/tests/batterystatstests/BatteryStatsViewer/res/values-zh-rCN/strings.xml:10]: I doubt that'd work. These strings aren't systemui

Comment (JohnsonRan) [reply core/tests/batterystatstests/BatteryStatsViewer/res/values-zh-rCN/strings.xml:10]: ehhhh... it's in settings... maybe `packages_apps_Settings`?

Comment (idoybh) [reply core/tests/batterystatstests/BatteryStatsViewer/res/values-zh-rCN/strings.xml:10]: nah. got it. will merge and amend my change for this soon

**Pull request merged**